src/thunderbots/software/util/visualizer_messenger/visualizer_messenger.cpp

```diff
@@ -11,20 +20,55 @@ namespace Util
         return vm;
     }
 
-    void VisualizerMessenger::initializePublisher(ros::NodeHandle node_handle)
+    void VisualizerMessenger::onConnection()
```

Comment:
This is a bit deceptive, just because this won't get called _on_ a connection, but rather it waits for any connection. As such, what about:
``` suggestion
    void VisualizerMessenger::receiveWebsocketConnections()
```

Reply:
Changed